enterprise/app/trends/percentile_chart.tsx

```diff
@@ -100,9 +115,11 @@ function PercentilesChartTooltip({
   extractP90,
   extractP95,
   extractP99,
+  triggerCallback,
 }: PercentilesChartTooltipProps) {
   if (!active) return null;
   let data = payload[0].payload;
+  triggerCallback(data);
```

Comment:
nit: IIRC the React docs strongly recommend against having `render()` do anything that can mutate state. Would it work to put this in a class component and maybe put this in `componentDidUpdate`?

Reply:
Sure yeah done